network/constants.go

```diff
@@ -0,0 +1,6 @@
+package network
+
+// Load balancing strategies.
+const (
+	RoundRobinStrategy = "ROUND_ROBIN"
+)
```

Comment:
All constants should reside in `config/constants.go` and should be referenced from there.

Reply:
Moved in `config/constants.go`